modules/nw-openshift-sdn-modes.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_sdn/about-openshift-sdn.adoc
+
+[id="nw-openshift-sdn-modes_{context}"]
+= OpenShift SDN network isolation modes
+
+ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
+OpenShift SDN provides three SDN modes for configuring the pod network:
+
+* The _network policy_ mode allows project administrators to configure their own
+isolation policies using NetworkPolicy objects. Network policy is the default mode in {product-title} {product-version}.
+
+* The _multitenant_ mode provides project-level isolation for pods and services.
+pods from different projects cannot send packets to or receive packets from pods
```

Comment:
s/send packets to or receive packets from/send or receive packets from/ ?

Reply:
@bobfuru, because it's networking, I'm inclined to leave it alone just in case.